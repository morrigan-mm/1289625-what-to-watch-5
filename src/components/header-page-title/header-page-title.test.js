import React from "react";
import renderer from "react-test-renderer";
import HeaderPageTitle from "./header-page-title";

describe(`HeaderPageTitle`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(
          <HeaderPageTitle>content</HeaderPageTitle>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
