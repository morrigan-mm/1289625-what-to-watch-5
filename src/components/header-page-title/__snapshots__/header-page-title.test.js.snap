// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderPageTitle Should render correctly 1`] = `
<h1
  className="page-title user-page__title"
>
  content
</h1>
`;
